src/services/consumer-view.ts

```diff
@@ -0,0 +1,95 @@
+import { getFileService } from '../utils/get-file-service';
+import { Revision } from '../entities/dataset/revision';
+import { validateParams } from '../validators/preview-validator';
+import { CSVHeader, ViewDTO, ViewErrDTO } from '../dtos/view-dto';
+import { FactTableColumnType } from '../enums/fact-table-column-type';
+import { DatasetDTO } from '../dtos/dataset-dto';
+import { Dataset } from '../entities/dataset/dataset';
+
+export const getTableRowsNoFilter = async (
+  datasetId: string,
+  revision: Revision,
+  lang: string,
+  startRow: number,
+  endRow: number,
+  sortBy?: string
+) => {
+  const { parquetMetadata, parquetQuery } = await import('hyparquet');
+  const fileService = getFileService();
+  const stream = await fileService.loadStream(`${revision.id}_${lang}.parquet`, datasetId);
```

Comment:
The fileService/datalake has a `loadBuffer()` method - can't we just use that instead of converting the stream?